wixcode-seo/wix-animations/TimeLine.service.json

```diff
@@ -0,0 +1,814 @@
+                      "  \"duration\": 1000",
+                      "})",
+                      "",
+                      "timeline2.add(myImage2, {",
+                      "  \"opacity\": 0,",
+                      "  \"duration\": 1000",
+                      "})" ],
+                  "extra":
+                    {  } },
+                { "title": "Add animations on multiple page elements together",
+                  "body":
+                    [ "import wixAnimations from 'wix-animations';",
+                      "",
+                      "let timeline = wixAnimations.timeline();",
+                      "",
+                      "// ...",
+                      "",
+                      "const myImage1 = $w(\"#myImage1\");",
+                      "const myImage2 = $w(\"#myImage2\");",
+                      "const vectorImages = $w(\"VectorImage\");",
+                      "",
+                      "// Adds an animation for myImage1 ",
+                      "// and myImage2 together",
+                      "timeline2.add([myImage1, myImage2], {",
+                      "  \"rotate\": 360,",
+                      "  \"scale\": .5,",
+                      "  \"duration\": 1000",
+                      "})",
+                      "",
+                      "// Adds an animation for all the",
+                      "// vector images on the page",
+                      "timeline2.add(vectorImages, {",
+                      "  \"opacity\": 0,",
+                      "  \"duration\": 1000",
+                      "})" ],
+                  "extra":
+                    {  } },
+                { "title": "Add animations to a timeline and play it",
+                  "body":
+                    [ "import wixAnimations from 'wix-animations';",
+                      "",
+                      "let timeline = wixAnimations.timeline();",
+                      "",
+                      "$w.onReady( function () {",
+                      "  const myImage1 = $w(\"#myImage1\");",
+                      "  const myImage2 = $w(\"#myImage2\");",
+                      "",
+                      "  timeline.add( [myImage1, myImage2], {",
+                      "    \"rotate\": 360,",
+                      "    \"scale\": .5,",
+                      "    \"duration\": 1000",
+                      "  } );",
+                      "",
+                      "  timeline.add( ",
+                      "    myImage, ",
+                      "    {",
+                      "      \"opacity\": 0,",
+                      "      \"duration\": 500",
+                      "    }, ",
+                      "    \"-=500\"",
+                      "  );",
+                      "",
+                      "  timeline.play();",
+                      "} );" ],
+                  "extra":
+                    {  } } ],
+            "extra":
+              {  } },
+        "extra":
+          {  } },
+      { "name": "onComplete",
```

Comment:
I guess you decided to make this present tense to match onStart and onRepeat?

Reply:
I didn't decide anything. Never when through spec review. Wasn't interested in making a fuss.